frontend/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="./bitbyte.png" />
```

Comment:
I think you removed the icon by mistake.

Reply:
This was a duplicate line, the icon is still present [here](https://github.com/BitByte-TPC/TPC-website/blob/376dab9696795d5f993e5882526b771384b5c760/frontend/index.html#L22-L26). And moreover, there is no file named `bitbyte.png`.

Reply:
The icon was not showing when I ran it in local. Maybe we have to change the path of the icon [here](https://github.com/BitByte-TPC/TPC-website/blob/376dab9696795d5f993e5882526b771384b5c760/frontend/index.html#L22-L26) to `/public/favicon.ico`

Reply:
It was working fine for me, but I've still changed the path to `/favicon.ico`. We don't need to prepend `public` to the path as Vite by itself looks into the `public` directory.